.github/workflows/main_wordlejb.yml

```diff
@@ -1,13 +1,15 @@
 # Docs for the Azure Web Apps Deploy action: https://github.com/Azure/webapps-deploy
 # More GitHub Actions for Azure: https://github.com/Azure/actions
 
-name: Build and deploy ASP.Net Core app to Azure Web App - Wordle2023
+
+name: Build and deploy ASP.Net Core app to Azure Web App - WordleJB
+
 
 on:
   push:
     branches:

```

Comment:
You can either add the assignment 3 branch to this, or can remove the branches component entirely to make this happen every time you push